### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by seanschroeder** [designbyfront/The-Goldilocks-Approach#6]
Doesn't look like anything has been updated since it was launched a year ago. Just curious if it's still active. Seems like a neat approach.

**Comment by chrisarmstrong** [comment]
Not dead, just resting :) We've been busy with http://typecastapp.com for the past year, but will definitely be continuing to support Goldie. Would love to hear any suggestions for improvements you have?

**Comment by seanschroeder** [comment]
Yep, I'm still waiting for my typecast invite ; ). Just discovered both of these. Glad to hear this isn't dead. I think this would make a great integration with Bootstrap. This approach would compliment it perfectly, as it would 320andup and skeleton. Any consideration of any of these? Keep up the great work.

**Comment by davidbgk** [comment]
@chrisarmstrong maybe merging existing pull-requests would be a good start?
